### DISCUSSION TITLE
购买了一年的专业版，但是无法通过邮箱登陆，无法使用专业版，请给我解决

### DISCUSSION THREAD

**Discussion by 172099366** [getlantern/lantern-forum-cn#778] [Q&A]
我成功购买了专业版，但是为什么不能通过邮箱登陆，始终无法使用。 帮助我尽快解决，我看大家好多都遇到了这样的问题，官方到底要怎么解决这个问题

我的邮箱yujb@siruinet.com
![Screenshot_20240812_222918_com eg android AlipayGphone](https://github.com/user-attachments/assets/f8c9eadf-6712-43c0-afb2-1b0f67b839ee)

**Comment by MaxMKMax** [comment]
@172099366 Okay! 我会帮您提交问题, 过 2 天请您查看这个帖子个帖子还是邮箱. 谢谢您支持蓝灯!

**Reply by 172099366** [reply]
@Mike777Khoo，请问有什么进展， 换个邮箱172099366@qq.com是否可以

**Reply by 172099366** [reply]
以前172099366@qq.com是可以收到邮箱

**Reply by 172099366** [reply]
这个问题能解决吗，不能解决退钱回来

**Reply by MaxMKMax** [reply]
@172099366 您好. 我已经发激活码到您的邮箱了. 请查看. 不好意思让您久等了. 真的非常抱歉. 谢谢您耐心的等待!

**Reply by MaxMKMax** [reply]
172099366@qq.com
发到这个邮箱

**Reply by 172099366** [reply]
@Mike777Khoo，请问有什么进展， 换个邮箱172099366@qq.com是否可以

**Comment by MaxMKMax** [comment]
@172099366 您好. 好的, 我帮您提交换邮箱的申请. 对了, 您的问题一直没有被解决吗? 还是解决了问题又复发了?

**Comment by MaxMKMax** [comment]
https://lantern.freshdesk.com/a/tickets/158115
已经帮您提交好了.

**Comment by 172099366** [comment]
去年是你帮我解决的，今年到期了又遇到同样的问题






小刀
***@***.***
